Classes/ContentObject/JsonContentContentObject.php

```diff
@@ -12,18 +12,62 @@
 namespace FriendsOfTYPO3\Headless\ContentObject;
 
 use FriendsOfTYPO3\Headless\Utility\HeadlessUserInt;
-use JsonException;
 use TYPO3\CMS\Core\Utility\GeneralUtility;
 use TYPO3\CMS\Frontend\ContentObject\ContentContentObject;
 use TYPO3\CMS\Frontend\ContentObject\ContentObjectRenderer;
 
 use function array_merge;
 use function count;
+use function is_array;
 use function json_decode;
 use function json_encode;
+use function json_last_error;
 use function strpos;
 use function trim;
 
+use const JSON_ERROR_NONE;
+
+/**
+ * CONTENT_JSON Content object behaves & has the same options as standard TYPO3' Content
+ * main difference is content is grouped by colPol field & encoded into JSON by default.
+ *
+ * CONTENT_JSON has the same options as CONTENT, also adds two new options for edge cases in json context
+ *
+ * ** merge ** option
+ * New option allows to generate another CONTENT_JSON call in one definition & then merge both results into one dataset
+ * (useful for handling slide feature of CONTENT cObject)
+ *
+ * for example:
+ *
+ * lib.content = CONTENT_JSON
+ * lib.content {
+ *    table = tt_content
+ *    select {
+ *        orderBy = sorting
+ *        where = {#colPos} != 1
+ *    }
+ *    merge {
+ *        table = tt_content
+ *        select {
+ *           orderBy = sorting
+ *           where = {#colPos} = 1
+ *       }
+ *       slide = -1
+ *    }
+ *  }
+ *
+ * ** doNotGroupByColPos = 0(default)|1 **
```

Comment:
why not `groupByColPos` and make `true` as default, so user can disable grouping, not other way around